src/types.mjs

```diff
@@ -0,0 +1,23 @@
+/**
+ * This file includes the types used in this project.
+ * Switching to TypeScript would be much better but here we are...
```

Comment:
You can use .d.ts files even when using JSDoc...

Reply:
Sure, but note that, this too, is only considered as a step towards transitionning to TypeScript.
It is even mentionned on the homepage of the TS website: from JS, to JS + JSDoc, to JS + typings (`.d.ts`), to finally TS.

And since we are only working with JS yet, I preferred to keep it coherent with the codebase (in JS). That makes it easier for JS-only devs to work with, without scaring them with TypeScript or typings files at all.

Reply:
I get that, although I feel like longer types in JSDoc just get messy